apps/ledger-live-mobile/src/newArch/features/FirmwareUpdate/components/UpdateBanner/ViewProps.tsx

```diff
@@ -0,0 +1,11 @@
+import { lastConnectedDeviceSelector } from "~/reducers/settings";
+
+export interface ViewProps {
```

Comment:
Open question, do we want to extract types into a dedicated file? This is something I kept next to the component in Market.

Reply:
I think it's useful so you can at the same time use it as return type from the `useViewModel` and props type for the view.
It makes it much easier then to do simple refactoring operations like renaming one of the props without breaking everything.
The alternative would be to define it in the same file as the `useViewModel` (because the component already imports from this file).